[2025-05-21T17:58:34.435+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-21T17:58:34.463+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-21T17:58:19.049154+00:00 [queued]>
[2025-05-21T17:58:34.472+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-21T17:58:19.049154+00:00 [queued]>
[2025-05-21T17:58:34.473+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-21T17:58:34.489+0000] {taskinstance.py:2327} INFO - Executing <Task(BranchPythonOperator): check_if_retrain_needed> on 2025-05-21 17:58:19.049154+00:00
[2025-05-21T17:58:34.497+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monitor_PWU', 'check_if_retrain_needed', 'manual__2025-05-21T17:58:19.049154+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/monitor_PWU.py', '--cfg-path', '/tmp/tmpfifxs1_s']
[2025-05-21T17:58:34.499+0000] {standard_task_runner.py:91} INFO - Job 60: Subtask check_if_retrain_needed
[2025-05-21T17:58:34.501+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=208) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-21T17:58:34.502+0000] {standard_task_runner.py:63} INFO - Started process 209 to run task
[2025-05-21T17:58:34.545+0000] {task_command.py:426} INFO - Running <TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-21T17:58:19.049154+00:00 [running]> on host 20faf461a3a5
[2025-05-21T17:58:34.638+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monitor_PWU' AIRFLOW_CTX_TASK_ID='check_if_retrain_needed' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T17:58:19.049154+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T17:58:19.049154+00:00'
[2025-05-21T17:58:34.639+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-21T17:58:34.654+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-05-21T17:58:34.666+0000] {python.py:237} INFO - Done. Returned value was: trigger_retrain
[2025-05-21T17:58:34.667+0000] {branch.py:36} INFO - Branch into trigger_retrain
[2025-05-21T17:58:34.667+0000] {skipmixin.py:178} INFO - Following branch trigger_retrain
[2025-05-21T17:58:34.681+0000] {skipmixin.py:246} INFO - Skipping tasks [('end', -1)]
[2025-05-21T17:58:34.709+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-21T17:58:34.730+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=monitor_PWU, task_id=check_if_retrain_needed, execution_date=20250521T175819, start_date=20250521T175834, end_date=20250521T175834
[2025-05-21T17:58:34.756+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-21T17:58:34.789+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-21T17:58:34.795+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
